.github/workflows/ci-test.yml

```diff
@@ -1266,51 +1081,3 @@ jobs:
         uses: ./.github/actions/early-terminator
         with:
           github-token: ${{secrets.GITHUB_TOKEN}}
-
```

Comment:
we should also remove the step `- name: Post test results on PR` from line 1068 which is used to post temp test result for comparison only

Reply:
Do you mean line 1253? we still want forge to post results to the PR with the perf numbers

Reply:
yea, sorry for a little bit confusion, there is an another step above also named Post test results on PR, that is the actual step to post perf number

Reply:
As a temporary step I assume it will be removed shortly, I might forgot to change the name 😄

Reply:
I think i removed the correct one now